api_tests/btsieve/no_ledger/test.ts

```diff
@@ -0,0 +1,85 @@
+import * as chai from "chai";
+import chaiHttp = require("chai-http");
+import { HarnessGlobal } from "../../lib/util";
+import { Btsieve } from "../../lib/btsieve";
+
+const should = chai.should();
+chai.use(chaiHttp);
+
+declare var global: HarnessGlobal;
+
+const btsieve = new Btsieve("main", global.config, global.project_root);
+
+setTimeout(async function() {
+    describe("Test btsieve API - no ledger connected", () => {
+        describe("BTsieve", () => {
+            describe("Ping", () => {
+                it("btsieve ping should respond with 200", async function() {
+                    return chai
+                        .request(btsieve.url())
+                        .get("/ping")
+                        .then(res => {
+                            res.should.have.status(200);
+                        });
+                });
+            });
+        });
+
+        describe("Bitcoin", () => {
+            describe("Transactions", () => {
+                it("btsieve should respond `SERVICE UNAVAILABLE` as ledger is not connected if queried for transaction", async function() {
```

Comment:
🙃 Could we have this test in the `ethereum_only` tests?

Reply:
My idea was to have _clean_ end-to-end tests in the `bitcoin/ethereum_only` tests and these edge cases outside. 
Do you see an issue with having these tests here?

Reply:
I was proposing to duplicate to confirm that indeed there is bitcoin service unavailable when only ethereum is available. Feel free to not action the comment.